monarch_mapping_commons.Makefile

```diff
@@ -8,18 +8,27 @@ MAPPING_DIR = 					mappings
 SCRIPT_DIR =					scripts
 SRC_DIR = 						sources
 TMP_DIR = 						tmp
+METADATA_DIR = 					metadata
 RUN = poetry run
 
 
 $(MAPPING_DIR)/ $(SCRIPT_DIR)/ $(SRC_DIR)/ $(TMP_DIR)/:
 	mkdir -p $@
 
-$(MAPPING_DIR)/mondo.sssom.tsv: | $(MAPPING_DIR)/
+$(MAPPING_DIR)/mondo.sssom.tsv:
+	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
```

Comment:
out of curiosity, why run mkdir on every single target, rather than having it as a dependency and checking that it already exists?

Reply:
Its a stylistic thing - it causes zero computation overhead, but makes for a less complex dependency tree.

This paradigm is _much_ more important when depending on `.PHONY` goals. previously we added phony dependencies to the makefile, but this causes make pipelines to _always_ be run again, which defeats the purpose of make. So, we decided that simplifying the dependency tree in general is the best option!